### DISCUSSION TITLE
Text not taking semibold (600) font weight in pango markup text

### DISCUSSION THREAD

**Discussion by Ruchika-K** [libvips/libvips#3037] [Q&A]
font_weight='600'  gets render in normal size.
But ,
font_weight='700'  gets render as quite bold

Is there some version issue?

**Comment by jcupitt** [comment]
Hi @Ruchika-K,

Increasing values of `font_weight` ought to get more bold, so that's probably working as intended. Do you mean `font_size`, perhaps?

https://docs.gtk.org/Pango/pango_markup.html

**Reply by Ruchika-K** [reply]
Hey,
I meant font_weight. There is a slight difference in these 2 values =600 and 700. 
Yeah it's working as expected.  Probably missed something earlier. 
Thanks though :)
